views/requests/view.erb

```diff
@@ -1 +1,33 @@
-    Requests I've received
\ No newline at end of file
+
+
```

Comment:
Maybe put the left and riht containers in the .centered-container div so they're in the middle of the page.

Reply:
It should work now.